jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchesTests.java

```diff
@@ -28,55 +34,110 @@ public void before() {
         redSwitch.uncheck();
         redSwitch.is().unchecked();
     }
 
+    @Test
+    public void flatSwitchesTest() {
+        Switch flatSwitch = flatSwitches.get(1);
+        flatSwitch.show();
+        flatSwitch.is().flat();
+        flatSwitch.is().notInset();
+        flatSwitch.is().lightTheme();
+        flatSwitch.has().notMessages();
+    }
+
+    @Test
+    public void insetSwitchesTest() {
+        Switch insetSwitch = insetSwitches.get(1);
+        insetSwitch.show();
+        insetSwitch.is().inset();
+    }
+
     @Test
     public void arraySwitchTest() {
+        modelAsArraySwitches.get(1).show();
         modelAsArraySwitches.get(1).is().checked();
         modelAsArraySwitches.get(2).is().unchecked();
-        modelArray.is().text("[ \"John\" ]");
+        modelArray.has().text("[ \"John\" ]");
         modelAsArraySwitches.get(2).check();
-        modelArray.is().text("[ \"John\", \"Jacob\" ]");
+        modelArray.has().text("[ \"John\", \"Jacob\" ]");
         modelAsArraySwitches.get(1).uncheck();
         modelAsArraySwitches.get(2).uncheck();
         modelArray.is().text("[]");
     }
 
     @Test
     public void modelAsBooleanSwitchesTest() {
+        modelAsBooleanSwitches.get(1).show();
         modelAsBooleanSwitches.get(1).is().enabled();
-        modelAsBooleanSwitches.get(1).has().hasLabel();
+        modelAsBooleanSwitches.get(1).has().label();
         modelAsBooleanSwitches.get(1).label().has().text(Matchers.containsString("true"));
         modelAsBooleanSwitches.get(1).uncheck();
-        modelAsBooleanSwitches.get(2).has().hasLabel();
+        modelAsBooleanSwitches.get(2).has().label();
         modelAsBooleanSwitches.get(2).label().has().text(Matchers.containsString("false"));
     }
 
     @Test
     public void statesSwitchTest() {
-        statesSwitches.get(3).has().hasNoLabel();
+        statesSwitches.get(3).show();
+        statesSwitches.get(3).has().notLabel();
         statesSwitches.get(3).is().checked().and().disabled();
-        statesSwitches.get(4).has().hasNoLabel();
+        statesSwitches.get(4).has().notLabel();
         statesSwitches.get(4).is().unchecked().and().disabled();
-        statesSwitches.get(6).has().hasNoLabel();
+        statesSwitches.get(6).has().notLabel();
         statesSwitches.get(6).is().unchecked();
         statesSwitches.get(6).find("*[role = 'progressbar']").is().displayed();
     }
 
     @Test
     public void labelSwitchTest() {
-        labelSwitches.has().hasLabel();
+        labelSwitches.show();
+        labelSwitches.has().label();
+        labelSwitches.has().labelText("Turn on the progress:");
         labelSwitches.is().unchecked();
         progressSpinnerLabelSwitches.has().hasNoLabel();
         progressSpinnerLabelSwitches.is().notSpinning();
         labelSwitches.check();
         progressSpinnerLabelSwitches.is().spinning();
     }
+
+    @Test
+    public void variousAttributesSwitchesTest() {
```

Comment:
Please divide tests for feathers (for example, test to check theme, test to check messages, test to check readonly, ....)

Reply:
Done